data/campaigns/Winds_of_Fate/units/Outrigger.cfg

```diff
@@ -0,0 +1,58 @@
+#textdomain wesnoth-wof
+
+[unit_type]
+    id=Outrigger
+    name= _ "Outrigger"
+    race=mechanical
+    image="units/outrigger.png"
+    hitpoints=50
+    movement_type=float
+    [movement_costs]
+        village=3
+    [/movement_costs]
+    [defense]
+        village=100
+    [/defense]
+    [resistance]
+        fire=110
+    [/resistance]
+    movement=6
+    experience=50
+    level=2
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=26
+    usage=null
+    description= _ "Cutting light and shallow through the water's surface, these raiding craft easily intercept the more substantial and seaworthy vessels of Wesnoth. While well armed and heavily crewed for their size, orcish Outriggers nevertheless rely on numbers to overcome well defended shore settlements and shipping.
+
+The shallow draft of the Outrigger's hulls allows it to be landed at coastal villages, though this makes it entirely vulnerable."
+    [attack]
+        name=boarders
+        description=_"boarders"
+        icon=attacks/sword-orcish.png
+        type=blade
+        range=melee
+        damage=10
+        number=2
+    [/attack]
+    [attack]
+        name=onager
+        description=_"onager"
+        icon=attacks/rock_thrown.png
+        type=fire
+        range=ranged
+        damage=16
+        number=1
+    [/attack]
+    [attack_anim]
+        [filter_attack]
+            name=onager
+        [/filter_attack]
+        missile_start_time=-50
```

Comment:
I think this is a typo, looks better with `missile_start_time=-500`

Reply:
Good catch, thanks!
It looks like early on I made some similar mistakes with other units as well, due to a poor understanding of the animation system. So I updated a number of unit animations along these lines.